styles.css

```diff
@@ -0,0 +1,406 @@
+*{
+  margin: 0;
+  padding:0;
+}
+body{
+  background-size:cover;
+}
+@media print{
+  body img{
+    display:none;
+  }
+}
+a{
+  text-decoration: none;
+}
+/* Mobile Styles */
+@media only screen and (max-width: 500px){
+  .container{
+    margin: 30px 50px;
+    display: grid;
+    grid-gap: 15px;
+  }
+  header{
+    width:100% !important;
+    background-image:url("images/header-image.jpg"),linear-gradient(to right,rgba(255,0,0,0),rgba(255,0,0,1));
+    background-size:cover !important;
+  }
+  h4{
+    padding: 2px !important;
+  }
+  nav{
+    width:100% !important;
+    height:25px !important;
+  }
+  .main-container{
+   margin: 3px !important;
```

Comment:
Plz no px. setting left and right margins to something like 10% would help create whitespace and center content.